src/components/Blocks/Hero/Hero.module.scss

```diff
@@ -5,17 +5,25 @@
     gap: 60px;
     justify-content: center;
     height: 300px;
-    margin: 60px;
+    margin: 200px 60px 60px;
```

Comment:
Do pixels conflict with responsive design?

Reply:
Probably yes. We will have a good amount of fine-tuning to get everything responsive. Pixels are used everywhere right now though, so I think this is good enough to merge and then revisit.